allennlp/data/dataset_readers/reading_comprehension/squad.py

```diff
@@ -29,6 +29,10 @@ class SquadReader(DatasetReader):
 
     Parameters
     ----------
+    multiparagraph : ``bool``, optional (default=``False``)
```

Comment:
Nit: I'd put parameters that are less likely to be used / less central to the class at the bottom.